Documentation/release-notes/4567.md

```diff
@@ -0,0 +1,18 @@
+### Preliminary JDK 11 Support
+
+Xamarin.Android can now use JDK 11 to build libraries and applications.
```

Comment:
> Visual Studio will start installing the JetBrains Runtime OpenJDK 11 by default in the same release that contains this Xamarin.Android change.

Almost certainly *not*, not for d16-7 P3 anyway, and I *really* doubt the likelihood of d16-7 *period*.  For starters, it's only *needed* if Build-tools r30 is used, and --at present! -- it's *not* required to target API-R/Android 11.

(Even if it *were* required, I rather doubt the number of people who will be *using* Android 11 in the preview this is released in.)

More importantly, the Android Designer doesn't currently work with JDK 11.  We *can't* change the default, even if we wanted to.

Additionally, I don't know what the *rest* of the VS "ecosystem" looks like, and whether it can be updated to JDK 11.

Finally, let's take a gander at API-29, shall we?  That didn't become the *default* `$(TargetFrameworkVersion)` until *d16-6*, ~9 *months* after a Xamarin.Android release containing a stable Android 10.0 binding.

I think there is *no chance at all* of migrating VS to JDK 11 "soon."*

*Definition of "soon" is deliberately undefined.''

> If it's not yet known when Visual Studio will start installing OpenJDK 11 by default…

Will do, though I think it'll be "moot."  One of the (random) ideas is that we'll *delete* the `Documentation/release-notes` contents when a release note has gone stable.  My guess is that XA d16-7 will be stable *long before* VS adds JDK 11, at which point this release note *will likely not exist*.

> If it is already known that Visual Studio will *not* install OpenJDK 11 by default

99.99% certain of this.

> it could be good to add a sentence about when users might want to go to the trouble of finding and installing JDK 11 themselves.

I'm not sure *I* can write this, not quickly.  The only reason I know of to want/need JDK11 support is for Android SDK Build-tools r30+, but *why* would anybody want to use that, *especially* considering that it's still a "release candidate"?  *I don't know* why this would be desirable.

I like the blurb you provided.